### PR TITLE
Data input fields 

### PR DESCRIPTION
**Pull request by acflinch**

### CODE REVIEW

**Review by BlueGreenDragon — APPROVED** [review]
Looks good.

File: code/frontend/src/components/MovieQueryFormComponent.js [hunk 1]
```diff
@@ -23,13 +23,25 @@ function MovieQueryFormComponent({ setData, setLoading }) {
     const handleSubmit = async (e) => {
         e.preventDefault();  // Prevent default form submission behavior
 
-        // Get input values from the dynamically loaded form
-        const input1 = document.getElementById('input1').value;
-        const input2 = document.getElementById('input2').value;
+        const data_request = {
+            // Get input values from the dynamically loaded form
+            rank : document.getElementById('rank').value;
```
Comment (jshilts657) [inline code/frontend/src/components/MovieQueryFormComponent.js]: you need to replace the ; -> ,

File: code/frontend/src/components/MovieQueryFormComponent.js [hunk 2]
```diff
@@ -23,13 +23,25 @@ function MovieQueryFormComponent({ setData, setLoading }) {
...
+            release_end : document.getElementById('release_end').value;
+            score : document.getElementById('score').value;
+            genre_select : document.getElementById('genre_select').value;
+            rating_select : document.getElementById('rating_select').value;
+            budget : document.getElementById('budget').value;
+            box_office : document.getElementById('box_office').value;
+            cast_select : document.getElementById('cast_select').value;
+            director_select : document.getElementById('director_select').value;
+            writer_select : document.getElementById('writer_select').value;
+        }
```
Comment (jshilts657) [inline code/frontend/src/components/MovieQueryFormComponent.js]: add semi-colon at end  like };

**Review by jshilts657 — CHANGES_REQUESTED** [review]
Just two small fixes,
Honestly my bad I should have caught them earlier

File: code/frontend/src/components/MovieQueryFormComponent.js (hunk 1 above)
Comment (jshilts657) [reply code/frontend/src/components/MovieQueryFormComponent.js]: you need to do this for all elements in the json. 
Look at an example json object. each element is separated by a comma not a semi-colon

**Review by jshilts657 — APPROVED** [review]

**Review by rzheng2019 — CHANGES_REQUESTED** [review]
I was only able to run the tests when I changed the package.json testMatch path on line 16 to "<rootDir>/src/tests/*.test.js", the front slash next to the * needs to removed to grab all ".test.js" files

**Review by rzheng2019 — APPROVED** [review]
Looks good now.

**Pull request merged**